README.md

```diff
@@ -0,0 +1,24 @@
+# Py Unittest Walkthrough
+
+This repository is designed to be a learning tool for learning the usage and best practices of the Python unittest package 
+
+
+## Usage 
+
+To see how it works, if you have python3x installed you can run individual tests by:
+
+    cd test
+    cd unit_tests
+    python3 utils_test.py
+    python3 core_test.py
+
+You can get code coverage by:
+
+cd test
+
+    ./run_coverage.sh
```

Comment:
We'll need more prerequisites to getting this command working.
```
❯ ./run_coverage.sh
./run_coverage.sh: line 9: coverage: command not found
./run_coverage.sh: line 10: coverage: command not found
```

Reply:
Yes this will have to be included in the setup / prereqs portion.